### ISSUE TITLE
`python-no-eval` invalid for pytorch models

### ISSUE THREAD

**Issue by Mahdi-Hosseinali** [asottile/pyupgrade#952]
Hi,
Thanks for the great work. This package has improved the quality of the legacy code I'm working with dramatically. I have a issue with a new project I'm working on. Pytorch ml models can have different behaviors during training and inference. When the training is done, they should be put in [evaluation mode](https://pytorch.org/docs/stable/generated/torch.nn.Module.html#torch.nn.Module.eval). pyupgrade complains about `python-no-eval. When the only pre-commit issue is the no-eval, I use `--no-verify` flag but I have to do this every time I make some changes to that file. I am not sure what is the best way to approach this.

**Comment by asottile** [comment]
pyupgrade doesn't do anything with eval or complain it just changes code

**Comment by asottile** [comment]
your princess is in another castle
